pkg/mq/amqp_publisher.go

```diff
@@ -44,7 +44,10 @@ func Publisher(conn *AMQPConnection, xchg string, setup AMQPSetupFunc) chan<- Pu
 					if !ok {
 						continue loopNewConnection
 					}
-					msg.Publishing.Timestamp = time.Now()
+					if msg.Publishing.Headers == nil {
+						msg.Publishing.Headers = amqp.Table{}
```

Comment:
The library's timestamp options is accurate to the second, the table can be set to be accurate to the millisecond. as the tracing service uses this timestamp to determine processing time and time in queue, it would almost always display 0 seconds. it can now display the duration in Milliseconds.